src/strategies/erc1155-all-balances-of/index.ts

```diff
@@ -0,0 +1,60 @@
+import { subgraphRequest } from '../../utils';
+
+export const author = 'fragosti';
+export const version = '0.1.0';
+
+export const SUBGRAPH_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/alexvorobiov/eip1155subgraph'
+};
+
+export async function strategy(
+  _space,
+  network,
+  _provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const eip1155OwnersParams = {
+    accounts: {
+      __args: {
+        where: {
+          id_in: addresses.map((a) => a.toLowerCase())
+        }
+      },
+      id: true,
+      balances: {
+        value: true,
+        token: {
+          registry: {
+            id: true
+          }
+        }
+      }
+    }
+  };
+  if (snapshot !== 'latest') {
+    // @ts-ignore
+    eip1155OwnersParams.accounts.__args.block = { number: snapshot };
+  }
+  try {
+    const result = await subgraphRequest(
+      SUBGRAPH_URL[network],
+      eip1155OwnersParams
+    );
+    return result.accounts.reduce((acc, val) => {
+      const relevantTokenBalances = val.balances.filter((balance) => {
+        const isRightAddress =
+          balance.token.registry.id === options.address.toLowerCase();
+        return isRightAddress;
+      });
+      acc[val.id] = relevantTokenBalances.reduce(
```

Comment:
The voters addresses must be checksumed (not lowercase), you can use the function "getAddress(address)" from ethers.js to do that example: https://github.com/snapshot-labs/snapshot.js/blob/aa5804cbd9f8745b3c227f6be1786a418142f8ab/src/strategies/staked-keep/index.ts#L39